client/src/components/atoms/AlertDialog.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from 'react'
+
+interface Props {
+    dialogText: string
+    onExit: () => void
+    onStay: () => void
+}
+
+const AlertDialog: FunctionComponent<Props> = (props: Props) => {
+    return (
+        <div className="alert-box">
```

Comment:
Nitpick: use class name `alert-dialog` instead of `alert-box`

Reply:
Updated.